tests/components/zha/test_api.py

```diff
@@ -138,7 +138,7 @@ async def device_groupable(hass, zigpy_device_mock, zha_device_joined):
 
 @pytest.fixture
 async def zha_client(hass, hass_ws_client, device_switch, device_groupable):
-    """Test zha switch platform."""
+    """Get zha websocket client."""
```

Comment:
```suggestion
    """Get ZHA WebSocket client."""
```

Reply:
Seems like a good idea.
I left ``zha`` lower-cased initially, as it's always lower-case in that class and I mainly tried to fix some actuals typos instead of lower/upper-case, but I guess I can change that as well at this point, haha.

So I should probably change everything from ``zha`` -> ``ZHA`` there then?